File: python/liczby23.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  liczby23.py

def liczby2():
    ile = 0 
    for i in range(1, 10):
        for j in range(10):
            if i != j: 
                print("{}{} ".format(i, j), end="")
                ile = ile + 1 
    return ile
    
    
def liczby3():
    ile = 0 
    for i in range(1, 10):
        for j in range(10):
            for k  in range(10):
                if i != j and i != k and j != k: 
                    print("{}{}{} ".format(i, j, k), end="")
                ile = ile + 1 
    return ile
        
        
def main(args):
    liczby3
    print("/n/nLiczb 2-cyfrowych:", liczby3())
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: cpp/liczby23.cxx
/*
 * liczby23.cxx
 */


#include <iostream>

int main(int argc, char **argv)

int liczby2() {
    int ile = 0;
    for(int i = 1; i < 10; i++) {
        for(int j = 1; j < 10; j++) {
            if(i != j) {
                cout << i << j << " ";
                ile++;
            }
        }
    }
    return ile
}

{
    cout << "liczb 2-cyfrowych: " << liczby2();
    
	return 0;
}

